pack.ts

```diff
@@ -69,13 +70,26 @@ pack.addSyncTable({
 
       // Media type filter
       const mediaTypeFilter = new photos.MediaTypeFilter(mediaType as types.MediaTypes);
+      // mediaTypeFilter.setType(mediaType as types.MediaTypes)
       filters.setMediaTypeFilter(mediaTypeFilter);
 
-      const { nextPageToken, mediaItems }: { nextPageToken?: string, mediaItems?: types.MediaItemResponse[] } = (await photos.mediaItems.search(filters, 100, (context.sync.continuation?.nextPageToken as string | undefined)))?.body ?? {};
+      // Feature filter (favorites)
+      if (favorite) {
+        const featureFilter = new photos.FeatureFilter(types.MediaFeature.Favorites)
+        filters.setFeatureFilter(featureFilter);
+      }
+
+      const response = await photos.mediaItems.search(
+        filters,
+        '',
+        100,
+        (context.sync.continuation?.nextPageToken as string | undefined)
+      )
```

Comment:
I don't like the way the params are passed here. Maybe put them in an object with props so it's more clear?

Reply:
Will be done in the next PR